src/Compilers/CSharp/Portable/Operations/CSharpOperationNodes.cs

```diff
@@ -28,6 +28,24 @@ internal sealed class CSharpLazyNoneOperation : LazyNoneOperation
     }
 
 
+    internal sealed class CSharpLazyNonePatternOperation : LazyNoneOperation, IPatternOperation
```

Comment:
Is this a placeholder until the IOperation part of the feature is done? If so consider adding a PROTOTYPE comment.

Reply:
The PROTOTYPE comments are in `CSharpOperationFactory` around line 293.

---
In reply to: [352880196](https://github.com/dotnet/roslyn/pull/39913#discussion_r352880196) [](ancestors = 352880196)